src/Orckestra.Media.AutoImageResizing/Properties/AssemblyInfo.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System.Reflection;
+using System.Runtime.CompilerServices;
+using System.Runtime.InteropServices;
+
+// General Information about an assembly is controlled through the following
+// set of attributes. Change these attribute values to modify the information
+// associated with an assembly.
+[assembly: AssemblyTitle("Orckestra.Media.AutoImageResizing")]
+[assembly: AssemblyDescription("")]
+[assembly: AssemblyConfiguration("")]
+[assembly: AssemblyCompany("Orckestra A/S")]
+[assembly: AssemblyProduct("Orckestra.Media.AutoImageResizing")]
+[assembly: AssemblyCopyright("Orckestra A/S")]
```

Comment:
There's a specific format for copyright strings

"Copyright © Orckestra A/S 2021"